web/html/src/manager/admin/config/monitoring-admin.js

```diff
@@ -84,7 +84,7 @@ const HelpPanel = (props) => {
       The server uses <a href="https://prometheus.io" target="_blank" rel="noopener noreferrer">Prometheus</a> exporters to expose metrics about your environment.
       </p>
       <p>
-      Refer to the <a href="/docs/index.html" target="_blank">documentation</a> to learn how to to consume these metrics.
+      Refer to the <a href="/docs/suse-manager/4.0/administration/pages/prometheus.html" target="_blank">documentation</a> to learn how to to consume these metrics.
```

Comment:
@cbbayburt Just to confirm, will the rewrite rule
```
RewriteCond %{DOCUMENT_ROOT}/docs/suse-manager/beta3/$1
```
be update with `4.0` instead of `beta3` ?

Reply:
Yes. But we'll skip "rc1"

Reply:
Ok. Removed `suse-manager/4.0`.